protobuf/build.gradle

```diff
@@ -6,10 +6,13 @@ description = 'gRPC: Protobuf'
 
 dependencies {
     compile project(':grpc-core'),
-            project(':grpc-protobuf-lite'),
             libraries.protobuf,
             libraries.guava,
             libraries.protobuf_util
+
+    compile (project(':grpc-protobuf-lite')) {
+        exclude group: 'com.google.protobuf', module: 'protobuf-lite'
```

Comment:
No, because that isn't represented in the pom and so isn't transitive, ironically (or non-ironically, since you said no to transitivity! :smile:). So it would cause things that depend on grpc-protobuf to still depend on both protobuf-java and protobuf-lite.

Also, excluding only the explicit dependency that causes problems also prevents accidentally excluding other necessary dependencies.